cdk/README.md

```diff
@@ -83,18 +83,19 @@ For advanced deployment scenarios using [CDK CLI](https://docs.aws.amazon.com/cd
 ```bash
 path/to/setup-tool deploy \
   --setup-name my-setup \
-  --aws-region <region> \ # (e.g. us-east-1)
+  --aws-region <region> \  #(e.g. us-east-1)
```

Comment:
Please wrap these formatting adjustments into the commit that introduced these comments. Makes for nicer readability in the git history. (Also is unrelated to the actual change in this commit).